config/database.docker.yml

```diff
@@ -0,0 +1,23 @@
+<%
+  mysql    = URI(ENV['MYSQL_URL'] || 'mysql://root@127.0.0.1:3306/samson_development')
```

Comment:
Sure -- there is already a ton of database.yml files there. Is config/database.samson.yml doing anything, or can we eliminate that one too while we're at it?